rosidl_generator_cpp/resource/msg.hpp.template

```diff
@@ -15,6 +15,7 @@
 #ifndef __@(spec.base_type.pkg_name)__@(subfolder)__@(get_header_filename_from_msg_name(spec.base_type.type))__hpp__
 #define __@(spec.base_type.pkg_name)__@(subfolder)__@(get_header_filename_from_msg_name(spec.base_type.type))__hpp__
 
+#include "@(spec.base_type.pkg_name)/@(subfolder)/@(get_header_filename_from_msg_name(spec.base_type.type))__meta.hpp"
```

Comment:
Should we move this line one down (same as in the service one) to indicate that it uses the struct?